stable/grafana/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: grafana
-version: 3.5.8
-appVersion: 6.2.4
+version: 3.5.9
+appVersion: 6.2.5
```

Comment:
this represents the `grafana` image tag, so no need to update this if you are not updating the image.

Reply:
finally I know what this is :laughing: